src/components/common/UserProfileEdit/UserAdvanceSettings.tsx

```diff
@@ -0,0 +1,243 @@
+  },
+  validate: {
+    id: 'users.UserProfileEdit.UserAdvanceSettings.validate',
+    defaultMessage: 'Validate',
+  },
+  snackbarSuccess: {
+    id: 'users.UserProfileEdit.UserAdvanceSettings.snackbarSuccess',
+    defaultMessage: 'Profile settings have been updated.',
+  },
+});
+
+interface FormValues {
+  [SlotKey.Metatransactions]: boolean;
+  [SlotKey.DecentralizedMode]: boolean;
+  [SlotKey.CustomRPC]: string;
+}
+
+const validationSchema = yup.object({
+  [SlotKey.Metatransactions]: yup.bool(),
+  [SlotKey.DecentralizedMode]: yup.bool(),
+  [SlotKey.CustomRPC]: yup.string().url(),
+});
+
+const displayName = 'users.UserProfileEdit.UserAdvanceSettings';
+
+const UserAdvanceSettings = () => {
+  const [showSnackbar, setShowSnackbar] = useState<boolean>(false);
+  useEffect(() => {
+    if (showSnackbar) {
+      const timeout = setTimeout(() => setShowSnackbar(true), 200000);
+      return () => {
+        clearTimeout(timeout);
+      };
+    }
+    return undefined;
+  }, [showSnackbar]);
+
+  const {
+    settings: {
+      metatransactions: metatransactionsSetting,
+      decentralizedModeEnabled,
+      customRpc,
+    },
+    setSettingsKey,
+  } = useUserSettings();
+
+  const onSubmit = useCallback(
+    (values) => {
+      Object.entries(values).forEach(
+        ([key, value]: [SlotKey, string | boolean]) =>
+          setSettingsKey(key, value),
+      );
+    },
+    [setSettingsKey],
+  );
+
+  const metatransasctionsToggleAvailable = canUseMetatransactions();
+
+  const metatransasctionsAvailable = metatransasctionsToggleAvailable
+    ? metatransactionsSetting
+    : false;
+
+  return (
+    <Form<FormValues>
+      initialValues={{
+        [SlotKey.Metatransactions]: metatransasctionsAvailable,
+        [SlotKey.DecentralizedMode]: decentralizedModeEnabled,
+        [SlotKey.CustomRPC]: customRpc,
+      }}
+      validationSchema={validationSchema}
+      onSubmit={onSubmit}
+    >
+      {({ isValid, values, submitForm }) => (
+        <div className={styles.main}>
+          <Heading
+            appearance={{ theme: 'dark', size: 'medium' }}
+            text={MSG.heading}
+            textValues={{
+              // @ts-ignore
+              learnMoreLink: (
+                <ExternalLink
+                  text={{ id: 'text.learnMore' }}
+                  href=""
+                  className={stylesAdvance.link}
+                />
+              ),
+            }}
+          />
+          <div className={stylesAdvance.toggleContainer}>
+            <Toggle
+              label={MSG.labelMetaTx}
+              labelValues={{
+                isOn: values[SlotKey.Metatransactions],
+              }}
+              name={SlotKey.Metatransactions}
+              disabled={!metatransasctionsToggleAvailable}
+            />
+            <QuestionMarkTooltip
+              tooltipText={MSG.metaTooltip}
+              /* @ts-ignore */
+              tooltipTextValues={{ br: <br /> }}
```

Comment:
Could you look if this `/* @ts-ignore */` can be fixed? It seems like `QuestionMarkTooltip` defines this prop as `SimpleMessageValues` but looking at the comments on the different types:
<img width="713" alt="image" src="https://user-images.githubusercontent.com/112586815/203409676-b27c3dbd-3daa-4488-9b2b-3a38cb6a9c0b.png">
And with the fact that `QuestionMarkTooltip` renders a `FormattedMessage` maybe this could be changed to `ComplexMessageValues` or `UniversalMessageValues` instead?